.teamcity/_self/projects/WPComTests.kt

```diff
@@ -184,6 +186,49 @@ fun editorTrackingBuildType( targetDevice: String, buildUuid: String, atomic: Bo
 fun jetpackPlaywrightBuildType( targetDevice: String, buildUuid: String, jetpackTarget: String = "wpcom-production" ): E2EBuildType {
 	val idSafeJetpackTarget = jetpackTarget.replace( "-", "_" );
 	val targetJestGroup = if (jetpackTarget == "remote-site") "jetpack-remote-site" else "jetpack-wpcom-integration";
+
+	val triggers: Triggers.() -> Unit = {
+		if (jetpackTarget == "wpcom-staging") {
```

Comment:
**comment** - TIL that you can use if/else to define triggers. This is really cool!